src/Microsoft.ML.FastTree/RandomForestClassification.cs

```diff
@@ -49,7 +49,7 @@ internal FastForestOptionsBase()
         }
     }
 
-    public sealed class FastForestClassificationModelParameters :
+    public sealed class FastForestMulticlassModelParameters :
```

Comment:
>FastForestMulticlassModelParameters [](start = 24, length = 35)

thank you for not making it FastForestMulticlassClassificationModelParameters :)